auto_updater.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+git ls-remote --exit-code fork > /dev/null 2>&1
```

Comment:
Can this block be wrapped by a named function?  Same goes for all of your other logical blocks.  Helps readers understand the purpose

Reply:
done